packages/venia-concept/src/components/Checkout/Receipt/receipt.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { func, shape, string } from 'prop-types';
+import classify from 'src/classify';
+import Button, { darkThemeClasses } from 'src/components/Button';
+import defaultClasses from './receipt.css';
+
+export const CONTINUE_SHOPPING_BUTTON_ID = 'continue-shopping-button';
+export const CREATE_ACCOUNT_BUTTON_ID = 'create-account-button';
+
+class Receipt extends Component {
+    static propTypes = {
+        classes: shape({
+            body: string,
+            footer: string,
+            root: string
+        }),
+        handleContinueShopping: func,
+        order: shape({
+            id: string
+        }),
+        handleCreateAccount: func,
+        reset: func
+    };
+
+    static defaultProps = {
+        order: {},
+        handleContinueShopping: () => {},
+        reset: () => {},
+        handleCreateAccount: () => {}
+    };
+
+    componentWillUnmount() {
+        this.props.reset();
+    }
+
+    createAccount = () => {
+        this.props.handleCreateAccount(this.props.history);
+    };
+
+    continueShopping = () => {
+        this.props.handleContinueShopping(this.props.history);
+    };
+
+    render() {
+        const {
+            classes,
+            order: { id }
+        } = this.props;
+
+        return (
+            <div className={classes.root}>
+                <div className={classes.body}>
+                    <h2 className={classes.header}>
+                        Thank you for your purchase!
+                    </h2>
+                    <div className={classes.textBlock}>
+                        Your order # is{' '}
+                        <span className={classes.orderId}>{id}</span>
+                        <br />
+                        We'll email you an order confirmation with details and
```

Comment:
Apostrophes (in JSX text) really mess with diffs. Consider either wrapping this text in a string or escaping the apostrophe as an HTML entity (`rsquo;`).